src/styles/blocks/catalog.scss

```diff
@@ -1,3 +1,5 @@
+$transition-duration:300ms;
```

Comment:
I think it must be in separate file for variables.

Reply:
Done